packages/react-native/examples/react-native-cli/ios/AwesomeProject.xcodeproj/project.pbxproj

```diff
@@ -498,7 +499,8 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)";
+				PRODUCT_BUNDLE_IDENTIFIER = "honeybadger.react-native-cli.example.--PRODUCT-NAME-rfc1034identifier-";
```

Comment:
This is an auto-generated file... I'm not totally sure why this changed but it seems to work fine.